filip/clients/ngsi_v2/cb.py

```diff
@@ -880,18 +881,39 @@ def get_subscription_list(self,
             self.log_error(err=err, msg=msg)
             raise
 
-    def post_subscription(self, subscription: Subscription) -> str:
+    def post_subscription(self, subscription: Subscription,
+                          update: Optional[bool] = False) -> str:
         """
         Creates a new subscription. The subscription is represented by a
         Subscription object defined in filip.cb.models.
 
-        Args:
-            subscription:
+        If the subscription already exists, the adding is prevented and the id
+        of the existing subscription is returned.
```

Comment:
Please elaborate how subscriptions are compared to each other. This is not trivial!